### DISCUSSION TITLE
Feature Request: BarChart: Show Legend value under every bar

### DISCUSSION THREAD

**Discussion by baur** [grafana/grafana#48796] [Dashboards, Panels & Visualizations]
It would be very nice if show Legend values under every bars on top of the values (see picture below)

![image](https://user-images.githubusercontent.com/2898361/167072782-56157b3f-b774-40f7-8045-e809c11b1525.png)

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
